frontend/packages/ux-editor/src/components/TaskNavigation/TaskCard.tsx

```
@@ -16,35 +24,69 @@
+
+  const contextButtons =
+    editCardsFeatureFlag || layoutSetModel.type === 'subform' ? (
+      <>
+        {editCardsFeatureFlag && (
+          <StudioButton
+            variant='tertiary'
+            onClick={(_: MouseEvent<HTMLButtonElement>) => {
+              setEditing(true);
+            }}
+          >
+            <PencilIcon /> {t('ux_editor.task_card.edit')}
```

ESLint

[error] 'PencilIcon' is not defined.

(react/jsx-no-undef)